BraveRewardsUI/de.lproj/Localizable.strings

```diff
@@ -622,6 +634,51 @@
 /* No comment provided by engineer. */
 "RewardsOptInPrefix" = "Machen Sie sich bereit für das nächste Internet.";
 
+/* No comment provided by engineer. */
+"SKUEnableRewardsBody" = "Um diesen Gegenstand zu kaufen, müssen Sie Token verdienen. Schalten Sie Brave Rewards ein, um zu verdienen.";
+
+/* No comment provided by engineer. */
+"SKUEnableRewardsButtonTitle" = "Brave Rewards aktivieren";
+
+/* No comment provided by engineer. */
+"SKUEnableRewardsDisclaimer" = "Durch Tippen auf „Brave Rewards aktivieren“ stimmen Sie den Nutzungsbedingungen von Brave Rewards zu.";
+
+/* The hyperlinked portion of the copy: 'By tapping Enable Brave Rewards, you agree to Brave Reward’s Terms of Service.' */
+"SKUEnableRewardsDisclaimerLink" = "Allgemeine Geschäftsbedingungen";
+
+/* No comment provided by engineer. */
+"SKUEnableRewardsTitle" = "Brave Rewards aktivieren, um fortzufahren.";
+
+/* '%@' becomes 'BAT' or 'BAP' based on region */
+"SKUPurchaseBuyWithBAT" = "Kaufen mit %@";
+
+/* No comment provided by engineer. */
+"SKUPurchaseCompletedBody" = "Viel Spaß beim Einkauf.";
+
+/* No comment provided by engineer. */
+"SKUPurchaseCompletedTitle" = "Sie sind fertig!";
+
+/* First '%@' becomes 'Buy with BAT' or 'Buy with BAP' based on region. Second '%@' becomes 'Brave’s Terms of Sale' */
+"SKUPurchaseDisclaimerInformation" = "Indem Sie auf '%1$@' klicken, erklären Sie sich einverstanden mit den %2$@.";
```

Comment:
This is the correct syntax for this, right?

Reply:
Probably should double check that this works (not sure if regular `String(format:)` API takes these locations into account. May have to switch over to the Bundle.localizedFormat or w/e its called

Reply:
@iccub Tested and it works correctly

Reply:
I tested and it works fine